apps/examples/st_things/README.md

```diff
@@ -1,17 +1,50 @@
-# st_things sample
+# light sample for st_things
 
-This is an example for a st_things.
+This is a light example for a st_things.
 
-## How to build
+1. [Create a cloud-connected device](https://smartthings.developer.samsung.com/develop/workspace/ide/create-a-cloud-connected-device.html)
+2. [Edit device info json file](#edit-device-info-json-file)
+3. [Build light sample](#build-light-sample)
+4. [Download(Flash)](#downloadflash)
+
+## Create a cloud-connected device.
+To connect the cloud, shoude be create device information on [devworkspace](https://devworkspace.developer.samsung.com/smartthingsconsole/iotweb/site/index.html).
+More details can find [here](https://smartthings.developer.samsung.com/develop/workspace/ide/create-a-cloud-connected-device.html).
+
+## Edit device info json file.
+1. Move into $TIZENRT_BASEDIR/apps/examples/st_things/light/contents/.
+
+	```bash
+	cd  [TizenRT]/apps/examples/st_things/light/contents/
+	```
+
+2. Copy to device_def.json to [TizenRT]/tools/fs/contents/.
+
+	```bash
+	cp device_def.json ../../../../../tools/fs/contents/
```

Comment:
There is a way to copy some file to somewhere by Makefile, not by user. When you add that way, you can reduce two steps. only 1st and 4th step needs. And because user wants to run this app, it makes sense that user modifies something inside this example folder.

Reply:
This doc explains way how to run light example without rt ide.
If the user use ide, he doesn't need to know why each files function.

So, we will consider your opinion after this release.

Reply:
how about chage "../../../../.." to [TizenRT]